README.md

```diff
@@ -5,27 +5,44 @@
 This service enables postgraduate candidates to apply for initial teacher
 training.
 
-## Dependencies
+## Table of Contents
+
+* [Dependencies](#dependencies)
+* [Prerequisites for development](#dev-prerequisites)
+* [Setting up the development environment](#dev-env-setup)
+* [Docker workflow](#docker-workflow)
+* [Webpacker](#webpacker)
+  * [Debugging Webpacker](#webpacker-debug)
+* [Documentation](#documentation)
+  * [Nomenclature](#documentation-nomenclature)
+  * [Domain Model](#documentation-domain-model)
+  * [Environment Variables](#documentation-env-vars)
+    * [Dockerfile](#documentation-env-vars-dockerfile)
```

Comment:
This table of contents is very useful. I find the third level of nesting makes it less easy to scan, though — do you agree? Happy to leave it in if not

Reply:
Happy to remove this if others agree. I don't particularly have a view either way.